docs/benchmarks/data-comparisson.md

```diff
@@ -255,7 +255,7 @@ Borko, H.
 .W
      Graduate library school study of abstracting should be more than a 
 how-to-do-it course.
-It should include general material on the characteristcs and types of abstracts,
+It should include general material on the characteristics and types of abstracts,
```

Comment:
> I will merge this and then commit a new version without the changes I think shouldn't be changed. For the future this would be easier with the review comments I think

Yes, many apologies!! It's mostly that as I'm reviewing I like to make changes as I go along and then reread the entire file as a whole. Comments don't really scratch that, but I think I got a system so I don't make more pull requests like these from now on. I'll avoid these in the future.